content/_data/changelogs/weekly.yml

```diff
@@ -8252,16 +8255,17 @@
     category: major rfe
     pull: 4823
     references:
-      - pull: 4823
+      - url: https://www.jenkins.io/blog/2020/07/23/windows-support-updates/#windows-service-management-changes-in-jenkins-2-248
+        title: changes summary
       - url: https://github.com/winsw/winsw/releases
-        title: Windows Service Wrapper changelog
+        title: full WinSW changelog
       - url: https://github.com/jenkinsci/windows-slave-installer-module/releases/tag/windows-slave-installer-2.0
         title: Windows Agent Installer 2.0 changelog
     authors:
     - NextTurn
     - oleg-nenashev
     message: |-
-      Update Windows Service Wrapper from 2.3.0 to 2.9.0.
+      Update Windows Service Wrapper (WinSW) from 2.3.0 to 2.9.0.
```

Comment:
Would it make sense to also reference the .NET Framework versions here?

Reply:
Why? There were no .NET Framework dependency changes between 2.3.0 and 2.9.0.

Reply:
We used to bundle 2.3.0 bundle for .NET Framework 2.0 in Jenkins. Now we bundle 3.9.0 for .NET Framework 4.0

Reply:
Agreed. It occurred to me that there were issues specific to .NET Framework 2.0/3.5, e.g., lowercased environment variables...

Reply:
```suggestion
      Update Windows Service Wrapper (WinSW) from 2.3.0 executable for  .NET Framework 2.0 to 2.9.0 for .NET Framework 4.0.
```